Lesson2/src/Lesson2.java

```diff
@@ -0,0 +1,57 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Lesson2 {
+
+
+    public static void main(String[] args) throws IOException {
+        System.out.println(sum(10, 20));
+        System.out.println(sum(Integer.MAX_VALUE, Integer.MAX_VALUE));
+        System.out.println(max(50, 49));
+        System.out.println(average(new int[]{1, 2, 3, 4, 5}));
+        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+        System.out.println(calculateHypotenuse());
+    }
+
+    public static int sum(int a, int b) {
+        return a + b;
+    }
+
+    public static int max(int a, int b) {
+        if (a > b) {
+            return a;
+        } else {
+            return b;
+        }
+    }
+
+    public static double average(int[] array) {
+        double sum = 0, result = 0;
```

Comment:
cool! now it works. Approved. For future development please declare every variable separate, like
double sum = 0;
double result = 0; 

It increases the readability of code